src/routes/api/flags/+server.ts

```diff
@@ -19,11 +19,11 @@ const flagsToArray = (flags: RemoteFlags) => {
 export const GET = (async (event) => {
   const userFlagsPromise = weaklyAuthenticate(event).then((authResult) => {
     if (authResult) {
-      return readPath(`/flags/user/${authResult.uid}`, []);
+      return readPath<RemoteFlags>(`/flags/user/${authResult.uid}`, {});
```

Comment:
This was defaulting to an empty array before and I think that is wrong, should have been an empty object.